eZ Platform/APIs/Languages/README.md

```diff
@@ -0,0 +1,70 @@
+# v2: Updated API Language behaviour
+
+## Objective
+
+Simplify usage of the product for developers, avoid common mistakes when dealing with multi language installs,
+and make sure to cover needs for future UI needs.
+
+### Use Cases
+The identified new behaviour:
+- As a Developer, I want to get API entities in prioritised language of the given SiteAccess,
+  so I don't need to specify language in API Service calls
+- As a Developer, I want to get API entities in one Translation at a time,
+  so I don't need to specify language or use helpers when using the API entities
+
+##### Related use cases
+- As an Editor, in UI I want to be able to select languages of API entities to view and edit translation independently
+  of the current SiteAccess, _so I don't need to deal with several variants of backend to do that_
```

Comment:
@andrerom - I don't completely understand this use case. Can you elaborate on this? My understanding is that we have content objects independent of siteAccess in our current implementation of ezplatform. How is this different from the current logic?